### ISSUE TITLE
ERROR: Could not get parameter "responseFormat"!

### ISSUE THREAD

**Issue by mecurc** [n8n-io/n8n#1144]
**Describe the bug**
When i try to retry workflow with http node i got error 

**To Reproduce**
1. Create a workflow that throws error on http node
2. Open executions
3. Retry

Error
```
ERROR: Could not get parameter "responseFormat"!
Error: Could not get parameter "responseFormat"!
    at getNodeParameter (/usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/NodeExecuteFunctions.js:184:15)
    at Object.getNodeParameter (/usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/NodeExecuteFunctions.js:375:24)
    at Object.execute (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/HttpRequest.node.js:575:37)
    at Workflow.runNode (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/src/Workflow.js:490:37)
    at /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:370:62
    at /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:451:15
    at /usr/local/lib/node_modules/n8n/node_modules/p-cancelable/index.js:61:11
    at new Promise (<anonymous>)
    at new PCancelable (/usr/local/lib/node_modules/n8n/node_modules/p-cancelable/index.js:31:19)
    at WorkflowExecute.processRunExecutionData (/usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:297:16)
```

**Expected behavior**
Workflow succefully retrying

**Environment (please complete the following information):**
 - OS: docker image
 - n8n Version 0.92.0

**Additional context**

**Comment by janober** [comment]
Thanks a lot for the report. Bug got fixed.

**Comment by janober** [comment]
Got released with n8n@0.93.0
